Bonsai.Core/Expressions/IncludeWorkflowBuilder.cs

```diff
@@ -24,6 +24,8 @@ namespace Bonsai.Expressions
     public sealed class IncludeWorkflowBuilder : VariableArgumentExpressionBuilder, IGroupWorkflowBuilder, INamedElement, IRequireBuildContext
     {
         const char AssemblySeparator = ':';
+        internal const string BuildUriPrefix = "::build:";
+        const string DefaultSerializerPropertyName = "Property";
```

Comment:
Something less generic than `Property` might be valuable here in case this somehow finds its way into an exception message or something. (Even something like `__Property__` would make it searchable.)